go/porcelain/deploy.go

```diff
@@ -239,13 +239,13 @@ func walk(dir string) (*deployFiles, error) {
 				return nil
 			}
 
-			o, err := os.Open(rel)
+			o, err := os.Open(path)
 			if err != nil {
 				return err
 			}
 
 			file := &file{
-				Name:   rel,
+				Name:   path,
```

Comment:
are you sure you want to change this one? if so, I don't think you need the `rel` variable anymore.